index.js

```diff
@@ -265,9 +266,22 @@ function argsToTokens(args, options) {
 
     ArrayPrototypePush(tokens, { kind: 'positional', index, value: arg });
   }
+
   return tokens;
 }
 
+/**
+ * Check if the given option that includes a default value
```

Comment:
This description seems to be the old behaviour. I think this routine is returning whether the default value should be set, rather than it has been set.

Reply:
I should have fixed the jsdoc